source/codegen/metadata/nidcpower/__init__.py

```diff
@@ -1,11 +1,18 @@
 from .functions import functions
+from .functions_addon import functions_override_metadata
 from .attributes import attributes
+from .attributes_addon import attributes_override_metadata
 from .enums import enums
+from .enums_addon import enums_override_metadata
 from .config import config
 
 metadata = {
     "functions" : functions,
     "attributes" : attributes,
     "enums" : enums,
     "config" : config
-}
\ No newline at end of file
+}
+
+metadata['functions'].update(functions_override_metadata)
```

Comment:
Perhaps this is too simplistic a method of merging?
I considered borrowing from nimi-python's codegen.

Reply:
Yeah, I think this should be fine. May need to come back to it if we start using override_metadata for more things and it gets more complex, but not sure on that and don't think it's worth pre-emptively addressing.

Historically, we've only used our _addon.py files for metadata_validation suppressions for certain things and not for overriding any of the main metadata: https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nifgen/enums_addon.py#L7

And going forward, in general I don't think we'd want to modify existing metadata through this mechanism but would rather just fix the source of it (fix generation of it elsewhere or hand-edit it in the regular `functions.py`).